File: Backend/mapping-dotnet/MHP.CodingChallenge.Backend.Mapping.Data/DTO/ArticleDto.cs
using System;
using System.Collections.Generic;

namespace MHP.CodingChallenge.Backend.Mapping.Data.DTO
{
    public class ArticleDto
    {
        public long Id { get; set; }
        public String Title { get; set; }
        public String Description { get; set; }
        public String Author { get; set; }
        public List<ArticleBlockDto> Blocks { get; set; }
    }
}

File: Backend/mapping-kotlin/src/main/kotlin/com/mhp/coding/challenges/mapping/models/dto/ArticleDto.kt
package com.mhp.coding.challenges.mapping.models.dto

import com.mhp.coding.challenges.mapping.models.dto.blocks.ArticleBlockDto

data class ArticleDto(
    var id: Long,
    var title: String,
    var description: String,
    var author: String,
    var blocks: Collection<ArticleBlockDto>
)
